charts/opensearch/templates/extraManifests.yaml

```diff
@@ -1,8 +1,25 @@
-{{ range .Values.extraObjects }}
+{{- /* Define a recursive function to convert structured data to a plain string YAML */ -}}
+{{- define "structuredToPlainString" -}}
```

Comment:
Could you add extraManifest in sample ci [values](https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch/ci/ci-values.yaml) so that it is run after every change to the chart to make sure we are not breaking anything.

Reply:
Will do on next commit.

Reply:
@TheAlgo Added, please review if that is what you were looking for.